### PR TITLE
chore(react-chart): add getting started guide

### PR DESCRIPTION
**Pull request by alexey-semikozov**

### CODE REVIEW

File: packages/dx-react-chart/docs/guides/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# React Chart - Getting Started
+
+## Overview
+Devextreme React Chart is a data visualization component with bar, line, stacked, area, scatter, pie and other series.
```
Comment (gsobolev) [inline packages/dx-react-chart/docs/guides/getting-started.md]: 'Devextreme' -> 'DevExtreme'
'with bar...' -> 'providing bar...'?

File: packages/dx-react-chart/docs/guides/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# React Chart - Getting Started
+
+## Overview
+Devextreme React Chart is a data visualization component with bar, line, stacked, area, scatter, pie and other series. 
+It has axis plugin for display scale and legend plugin for a description of series. 
+Now, Devextreme React Chart is the stateless component and depends on properties and working in a controlled mode only.
```
Comment (gsobolev) [inline packages/dx-react-chart/docs/guides/getting-started.md]: 'Devextreme' -> 'DevExtreme'

> component and depends on properties and working in a controlled mode only

Can't get much sense. Do you mean: "component **that** depends on properties and work**s** in the controlled mode only"?

File: packages/dx-react-chart/docs/guides/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Polyfills
+
+React Chart uses the latest web platform standards, and cannot support older browsers like IE11 and Android 4. Use the ES2015 (ES6) polyfill to support these browsers. We recommend [BABEL Polyfill](https://babeljs.io/docs/usage/polyfill/), but you can use an alternative.
+
+## Using Chart component
+
+The Chart renders nothing by default. The root Chart component's nested plugin components implement its functionality, and it is necessary to specify at least one plugin that visualizes the chart data.
+
+Use the *Series, *Axis plugins to display the data as a simple chart:
```
Comment (gsobolev) [inline packages/dx-react-chart/docs/guides/getting-started.md]: Please double check styling here (the whole line is in cursive except the first two words).

File: packages/dx-react-chart/docs/guides/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+React Chart can work in other browsers if they use the latest version of WebKit, Blink, or Gecko, whether directly or via the platform's web view API. Some of these browsers may require adding [polyfills](#polyfills). However, such browsers, including Internet Explorer, may not work correctly.
+
+## Polyfills
+
+React Chart uses the latest web platform standards, and cannot support older browsers like IE11 and Android 4. Use the ES2015 (ES6) polyfill to support these browsers. We recommend [BABEL Polyfill](https://babeljs.io/docs/usage/polyfill/), but you can use an alternative.
+
+## Using Chart component
+
+The Chart renders nothing by default. The root Chart component's nested plugin components implement its functionality, and it is necessary to specify at least one plugin that visualizes the chart data.
```
Comment (gsobolev) [inline packages/dx-react-chart/docs/guides/getting-started.md]: '...Chart component's nested plugin components...' - This looks overcomplicated.

File: packages/dx-react-chart/docs/guides/getting-started.md (hunk 3 above)
Comment (alexey-semikozov) [reply packages/dx-react-chart/docs/guides/getting-started.md]: "*" mean any. *Series === [any]Series or [Line || Area || Spline]Series.
Yes, I really forget backslash char before "*" char.

**Pull request merged**